src/main/steps/applicant1/review-the-application/content.ts

```diff
@@ -11,7 +11,22 @@ import { enConnectionBulletPointsSummarisedForAllUsers } from '../../../app/juri
 import { jurisdictionMoreDetailsContent } from '../../../app/jurisdiction/moreDetailsContent';
 import { CommonContent } from '../../common/common.content';
 
-const en = ({ isDivorce, userCase, partner, userEmail, isApplicant2, isJointApplication }: CommonContent) => ({
+const displayCorrectAddress = (applicant: 'applicant1' | 'applicant2', userCase: Partial<CaseWithId>) => {
+  const isSolicitorsAddress = `${applicant}${userCase[`${applicant}SolicitorAddress`]?.trim() ? 'Solicitor' : ''}`;
```

Comment:
The name suggests it's a Boolean, but it's a string right

Reply:
yes its a string, i dont think it suggests a boolean. if you think so what name would you prefer?

Reply:
isXYZ kind of suggests boolean, maybe remove the is

Reply:
ah my bad, i was looking at displayCorrectAddress